cli/core/hack/apt/build_package.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ $(uname) != "Linux" ] || [ -z "$(command -v apt)" ]; then
+   echo "This script must be run on a Linux system that uses the APT package manager"
+   exit 1
+fi
+
+# VERSION should be set when calling this script
+if [ -z "${VERSION}" ]; then
+   echo "\$VERSION must be set before calling this script"
+   exit 1
+fi
+
+# Strip 'v' prefix as an apt package version must start with an integer
+VERSION=${VERSION#v}
+
+BASE_DIR=$(cd $(dirname "${BASH_SOURCE[0]}")/..; pwd)
+OUTPUT_DIR=${BASE_DIR}/apt/_output
+
+# Install build dependencies
+apt-get update
+apt-get install -y dpkg-dev curl
+
+# Copy the tanzu binary to the package directory
+mkdir -p ${OUTPUT_DIR}/tanzu-cli_${VERSION}_linux_amd64/usr/bin
+cp ${BASE_DIR}/../../../artifacts/linux/amd64/cli/core/v${VERSION}/tanzu-core-linux_amd64 ${OUTPUT_DIR}/tanzu-cli_${VERSION}_linux_amd64/usr/bin/tanzu
```

Comment:
How hard would it be to make this multi-arch?
Eventually we will want Graviton support for example.